src/internal/pachsql/table_info_test.go

```diff
@@ -0,0 +1,44 @@
+package pachsql_test
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/dockertestenv"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/internal/require"
+	"github.com/pachyderm/pachyderm/v2/src/internal/testsnowflake"
+)
+
+func TestGetTableInfo(t *testing.T) {
+	type testCase struct {
+		Name  string
+		NewDB func(t testing.TB) *pachsql.DB
+	}
+	tcs := []testCase{
+		{
+			Name:  "Postgres",
+			NewDB: dockertestenv.NewPostgres,
+		},
+		{
+			Name:  "MySQL",
+			NewDB: dockertestenv.NewMySQL,
+		},
+		{
+			Name:  "Snowflake",
+			NewDB: testsnowflake.NewSnowSQL,
+		},
+	}
+	ctx := context.Background()
+	for _, tc := range tcs {
+		t.Run(tc.Name, func(t *testing.T) {
+			db := tc.NewDB(t)
+			require.NoError(t, pachsql.CreateTestTable(db, "test_table"))
+			info, err := pachsql.GetTableInfo(ctx, db, "test_table")
+			require.NoError(t, err)
+			t.Log(info)
```

Comment:
do we plan to actually check correctness here at some point?

Reply:
Great catch! I found out by checking the values, that different databases return the order and values slightly different. I will add checks.